streams/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import org.modelix.MODELIX_JDK_VERSION
+
+plugins {
+    `maven-publish`
+    kotlin("multiplatform")
```

Comment:
We should use `` `modelix-kotlin-multiplatform` `` and remove lines 9-27 to use the multiplatform configuration from the convention.